src/renderer/dx/DxRenderer.cpp

```diff
@@ -15,7 +15,7 @@
 #pragma hdrstop
 
 static constexpr float POINTS_PER_INCH = 72.0f;
-static constexpr std::wstring_view FALLBACK_FONT_FACE = L"Consolas";
+static constexpr std::wstring_view FALLBACK_FONT_FACES[] = { L"Consolas", L"Lucida Console", L"Courier New" };

```

Comment:
i [nit-level] wish that these, the only code changes _not_ related to being a UWP, were a separate change... but I am 100% not going to block or hold up anything for that.

Reply:
To reduce my mental load, I'm not going to fork it off this time.